src/frontend/sharedComponents/BottomSheet/Content.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from "react";
+import { StyleSheet, View } from "react-native";
+import { TouchableHighlight } from "@gorhom/bottom-sheet";
+
+import { LIGHT_BLUE, MAGENTA, MAPEO_BLUE, RED, WHITE } from "../../lib/styles";
+import Button from "../Button";
+import Text from "../Text";
+
+interface BaseActionButtonConfig {
+  onPress: () => void;
+  text: React.ReactNode;
+  variation: "filled" | "outlined";
+}
+
+interface PrimaryActionButtonConfig extends BaseActionButtonConfig {
+  dangerous?: boolean;
```

Comment:
open to changing the terminology to something like `destructive` if that's clearer